src/views/SpaceSettings.vue

```diff
@@ -497,6 +520,38 @@ watchEffect(async () => {
               </div>
             </div>
           </Block>
+          <Block :title="$t('settings.voting')">
+            <UiInput v-model="votingDelay" :number="true" placeholder="12">
```

Comment:
Can you use `e.g. 1` days  and  `e.g. 5` days as placeholder?
Instead of `12` hours  and  `5` days